spt/OrangeBox/spt-serverplugin.cpp

```diff
@@ -904,4 +928,77 @@ CON_COMMAND(y_spt_find_seam_shot, "y_spt_find_seam_shot [<pitch1> <yaw1> <pitch2
 
 	Msg("Could not find a seam shot. Best guess: setang %.8f %.8f 0\n", test.x, test.y);
 }
+
+
+CON_COMMAND(tas_script_load, "Loads and executes an .stas script. Usage: tas_load_script [script]")
+{
+#if defined( OE )
+	if (!engine)
+		return;
+
+	ArgsWrapper args(engine.get());
+#endif
+
+	if (args.ArgC() > 1)
+	{
+		scripts::g_TASReader.ExecuteScript(args.Arg(1));
+	}
+	else
+		Msg("Loads and executes an .stas script. Usage: tas_load_script [script]\n");
+
+
+}
+
+CON_COMMAND(tas_script_search, "Starts a variable search for an .stas script. Usage: tas_load_search [script]")
+{
+#if defined( OE )
+	if (!engine)
+		return;
+
+	ArgsWrapper args(engine.get());
+#endif
+
+	if (args.ArgC() > 1)
+	{
+		scripts::g_TASReader.StartSearch(args.Arg(1));
+	}
+	else
+		Msg("Starts a variable search for an .stas script. Usage: tas_load_search [script]\n");
+
+
+}
+
+CON_COMMAND(tas_script_result_success, "Returns a successful result in a variable search.")
```

Comment:
I'd change these two to something like `Signals that the current search result was successful`